CONTRIBUTORS.md

```diff
@@ -0,0 +1,99 @@
+# Keycloak Community
+
+Keycloak is an Open Source Identity and Access Management solution for modern Applications and Services.
+
+## Building and working with the codebase
+
+Details for building from source and working with the codebase are provided in the [building and working with the code base](docs/building.md) guide.
+
+## Contributing to Keycloak
+
+Keycloak is an Open Source community driven project and we welcome contributions as well as feedback from the community.
```

Comment:
I just googled for it and believe it's "community-driven".

Reply:
Changed